docs/pages/faq.md

```diff
@@ -16,7 +16,7 @@ title: Frequently Asked Questions
 
 #### __What is a DC/OS service?__
 
-  A [DC/OS service](https://dcos.io/docs/latest/overview/concepts/#dcos-service) is an instance of a service in a DC/OS cluster.
+  A [DC/OS service](https://dcos.io/docs/latest/overview/concepts/#dcos-service) is an instance of an existing service (typically a stateful, distributed service) in a DC/OS cluster. DC/OS services built using the DC/OS SDK implement schedulers which allow for orchestrated installation, update and backup/recovery.
```

Comment:
By "existing," you mean something like Apache Spark, which has an independent existence outside of the DC/OS context? What about people creating SDK services? Do they exist independently of DC/OS?

Suggest: "DC/OS services built using the DC/OS SDK implement schedulers, which all for orchestrated installation, update, and backup/recovery."